evap/staff/templates/staff_course_form.html

```diff
@@ -85,7 +85,7 @@
 
```

Comment:
nice misunderstanding :)

from looking at the code, i'd expect those to show up: `{% if state == "new" ...` but they wouldn't make any sense, right. the question is whether you made sure they don't show up.